src/project-config.ts

```diff
@@ -14,12 +14,27 @@ import * as logging from 'plylog';
 
 const logger = logging.getLogger('cli.project-config');
 
+export const defaultSourceGlobs = [
+  'src/**/*',
+  // NOTE(fks) 06-29-2016: `polymer-cli serve` uses a bower.json file to display
+  // information about the project. The file is included here by default.
+  'bower.json',
+];
+
 export interface ProjectConfigOptions {
   root?: string;
   entrypoint?: string;
   shell?: string;
   fragment?: string[];
   fragments?: string[];
+
+  // @NOTE(fks) 08-23-2016: The options below were forked by polymer-build.
```

Comment:
Whoops, old habits. That's something I've used in past projects. 

fwiw tagging a comment with a date and person can be super useful for comments related to tech-debt or that are time sensitive that someone may want to follow up on months/years later.